### ISSUE TITLE
ESLint fix option silently fails

### ISSUE THREAD

**Issue by andyrooger** [TypeStrong/fork-ts-checker-webpack-plugin#400]
## Current behavior
With eslint running via `fork-ts-checker-webpack-plugin`, turning on `fix` stops all issues from being reported and doesn't fix anything. If I turn `fix` off, I get issues reported again.

## Expected behavior
With `fix` turned on fixable issues should be fixed. At least the unfixable issues should be reported.

## Steps to reproduce the issue
Set up `fork-ts-checker-webpack-plugin` to run eslint with `fix` on.

```js
new ForkTSCheckerWebpackPlugin({
    eslint: true,
    eslintOptions: {
        fix: true,
    }
})
```

Example `.eslintrc.js`:

```js
module.exports = {
    root: true,
    parser: '@typescript-eslint/parser',
    parserOptions: {
        tsconfigRootDir: __dirname,
        project: ['./tsconfig.json'],
        sourceType: 'module',
    },
    plugins: [
        '@typescript-eslint',
    ],
    extends: [],
    rules: {
        '@typescript-eslint/no-unused-vars': ['error],
        'semi': ['error'],
    }
};
```

Add a file like this for webpack to build:

```js
const x = 1

export default null;
```

Build with webpack. No issues reported.
Turn off `fix` in `eslintOptions` and both rules report issues.

## Environment
 - **fork-ts-checker-webpack-plugin**: 4.1.1
 - **typescript**: 3.3.4000
 - **eslint**: 6.8.0
 - **webpack**: 4.39.2
 - **os**: Windows 10

**Comment by andyrooger** [comment]
Seems on [this line](https://github.com/TypeStrong/fork-ts-checker-webpack-plugin/blob/4013a3f56839dd27e4df6a410ff20c90d56a7b97/src/createEslinter.ts#L27) we're calling `outputFixes` on the linter itself rather than `CLIEngine` which the eslint docs suggest. The error then seems to be swallowed below. If I change this locally everything works as expected.
